controller/src/main/java/com/softserve/borda/controllers/AuthController.java

```diff
@@ -63,4 +71,9 @@
         }
         throw new CustomAuthenticationFailedException();
     }
+
+    @GetMapping("/usernames/{username}")
+    public ResponseEntity<Boolean> checkUserExistingByUsername(@PathVariable String username) {
```

Comment:
For what does it serve?

Reply:
This is required for Front-End validation:
In registration page its checks if the given username is taken already. Because username should be unique.